examples/appendix/templates/login.html

```diff
@@ -0,0 +1,25 @@
+{% extends "templates/login.html" %}
+{% block site %}
+
+<div id="ipython-main-app" class="container">
+  <h1>You can't access this Binder</h1>
```

Comment:
I would like to iterate on the text a bit in a follow up PR for end user clarity/usability.